pyproject.toml

```diff
@@ -93,6 +93,7 @@ all_extras = [
   "matplotlib>=3.3.2",
   "mne",
   'numba<0.60,>=0.53',
+  'peft>=0.10.0',
```

Comment:
should this go in the `dl` dependency set instead?

Reply:
my thought process behind this was that the deep learning modules, particularly `HFTransformersForecaster` can be used without `peft`; it is required only when a specific fine-tuning approach is needed

Reply:
ok I understand, then I should move that to `dl`

another questions: It is mandatory to have `peft` in the dependencies becuase we need that for tests in CI, right? for that reason we cannot avoid `peft` completetly in `pyproject.toml`.

Reply:
Correct. `dl` dependency is being tested as part of old CI, so `peft` will be tested there as well.

We were discussing how to handle deep learning dependencies better, but it seemed too complex and I think it's gone out of planning for now.

Reply:
The errors in the CI pipeline are also probably caused by that line. Since due to that the bounds defined in dl set for transformers is not active and the newest version of transformers is installed and this leads to problems with the newest keras version